### ISSUE TITLE
does not work

### ISSUE THREAD

**Issue by Pererozhdennyy** [Jpedrocs/OpenCvBot#1]
![image](https://user-images.githubusercontent.com/62129215/156423317-a3d6df70-cda0-43b7-b319-3aef0c166f04.png)

**Comment by Jpedrocs** [comment]
True, it is not working because the game offsets changes frequently, so i need to do some updates on the code to dump the csgo automatically when started the algorithm. I will try to fix it asap.
